src/templates/remove-exclusive-edit-mode-modal.html

```diff
@@ -0,0 +1,26 @@
+{# Modal for confirming the removal of exclusive edit mode when enforced #}
+{% if mode == 'edit' %}
+  {% set label = 'You have enabled "Enforce exclusive edit mode". Unlocking this entry will redirect you to view mode. If you enter edit mode, exclusive edit will be reactivated.'|trans %}
```

Comment:
make a single label with "You have enabled "Enforce exclusive edit mode".  If you enter edit mode, exclusive edit will be reactivated." and if you're on edit, add "Unlocking this entry will redirect you to view mode.". This way we do not duplicate the translated strings.